tests/integration/test_orders_cli.py

```diff
@@ -471,10 +455,129 @@ def test_cli_orders_create_clip(invoke,
             }
         }]
     }
+    assert order_request == json.loads(result.output)
+
+
+# TODO: add tests of "create --pretty" (gh-491).
+@pytest.mark.parametrize(
+    "id_string, expected_ids",
+    [('4500474_2133707_2021-05-20_2419', ['4500474_2133707_2021-05-20_2419']),
+     ('4500474_2133707_2021-05-20_2419,4500474_2133707_2021-05-20_2420',
+      ['4500474_2133707_2021-05-20_2419', '4500474_2133707_2021-05-20_2420'])])
+@respx.mock
+def test_cli_orders_create_basic_success(expected_ids,
+                                         id_string,
+                                         invoke,
+                                         order_description,
+                                         write_to_tmp_json_file):
+    mock_resp = httpx.Response(HTTPStatus.OK, json=order_description)
+    respx.post(TEST_ORDERS_URL).return_value = mock_resp
```

Comment:
Not high priority, but it would be cool if we could find a way to hoist these lines up above the body of the function. For clarity. I don't know enough about respx to see how to do it, though.